C++/Indexed Reference List.tmPreferences

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>
+        source.c constant.other.macro.c,
+        source.c support.type.c,
```

Comment:
What will `support.type` handle here?

Reply:
It handles all the references to library/user defined types.